src/TabSwitcher/Panes/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import { Container } from './elements';
+
+const { bool, node, number, string } = PropTypes;
+
+/**
+ * Panes hold the content of interactive tabs
+ *
+ * @param {number} activeIndex // index of the active tab
+ * @param {string} className /// className needed by styled components
+ *
+ * @return {jsx}
+ */
+class Panes extends PureComponent {
```

Comment:
Why you don't use a function here instead of a class? If you need only the things that need to be rendered are rendered, you can use [Memo](https://reactjs.org/docs/react-api.html#reactmemo) API from react